armi/reactor/components/__init__.py

```diff
@@ -154,7 +157,7 @@ def getComponentArea(self, cold=False):
         Parameters
         ----------
         cold : bool, optional
-            Compute the area with as-input dimensions instead of thermally-expanded
+            Ignored for this component
```

Comment:
I think the problem is that the mass of this component goes up if Tinput and Thot are different based on what Chris showed. This could be pretty unexpected and dangerous. Could we protect against having mass changing?

Reply:
Chris and I are talking about that for a separate ticket. For sure.